requirements/dev.txt

```diff
@@ -8,7 +8,7 @@ mypy
 nbqa
 pytest
 pylint
```

Comment:
I've removed `requirements/dev.txt` now. The development rule now would be to run `tox -e pre-commit` rather than using these packages from the local virtualenv.